lxd/api_cluster.go

```diff
@@ -47,6 +48,21 @@ import (
 	"github.com/lxc/lxd/shared/version"
 )
 
+type evacuateStopFunc func(inst instance.Instance) error
+type evacuateMigrateFunc func(s *state.State, r *http.Request, inst instance.Instance, targetMemberInfo *db.NodeInfo, live bool, startInstance bool, metadata map[string]any, op *operations.Operation) error
+
+type evacuateOpts struct {
+	s               *state.State
+	gateway         *cluster.Gateway
+	r               *http.Request
+	instances       []instance.Instance
+	mode            string
+	nodeName        string
```

Comment:
Can you call this srcMemberName for clarity (that its the source that is being evacuated and that we tend to use member rather than node for talking about cluster servers).